src/types/schema.ts

```diff
@@ -57,7 +57,7 @@ export interface FormSection {
 }
 
 export interface FormField {
-  label: string;
+  label?: string;
```

Comment:
@samuelmale this was changed to be optional because markdown type inputs don't have a label. Does this seem okay?

Reply:
That makes sense to me.